test/optimise.jl

```diff
@@ -102,6 +102,12 @@ using IntervalOptimisation: numeric_type
 
         end
 
+        @testset "Non smooth function in 2D" begin
+            global_min, minimisers = minimise( X -> ( (x,y) = X; abs(x-y) + abs(x) ), (-1..1) × (-1..1), structure = Structure )
+            @test global_min ⊆ 0..0 # it must return the exact minimum (if mid is used to find the first global_min)
```

Comment:
I think you should test the equality here (`global_min == 0..0`), because for example `emptyinterval() ⊆ 0..0` evaluates to true. So if for some weird bug global_min were empty, the test would not detect it in the current form.

Reply:
Ok, I'll modify it. It is ok to make a force push or do I add a new commit?

Reply:
Maybe a new commit might be better? They are going to be squashed into a single commit anyway when the PR is merged.

Reply:
Ok, ok.

Reply:
Done! :D